docs/reference/run.md

```diff
@@ -1207,21 +1207,25 @@ The health status is also displayed in the `docker ps` output.
 
 ### User
 
-`root` (id = 0) is the default user within a container. The image developer can
-create additional users. Those users are accessible by name.  When passing a numeric
-ID, the user does not have to exist in the container.
+The default user within a container is `root` (id = 0). You can set a default
```

Comment:
nit: `id` should probably be `uid` (and maybe mention `gid`


No need to change in this PR; looks like we should ideally also have some mention of "windows containers", where all of this will be quite different (not sure now from the top of my head if we even support `--user` for native Windows containers).